src/style.css

```
@@ -1 +1,20 @@
-/* styles go here */
+.form-field {
+  margin: 10px;
+}
```

use one-directional margins
```suggestion
.form-field {
  margin-bottom: 10px;
}
```